samples/batch/batch-ingestion-client/Setup/guide.md

```diff
@@ -2,11 +2,11 @@
 
 Get your audio files automatically transcribed without writing any code what-so-ever! The Batch Ingestion Client will monitor your dedicated Azure Storage container so that new audio files are transcribed automatically as soon as they land.
 
-The simplest way to explain this tech is as a zero-touch transcription solution for all your audio files in your Azure Storage. If you are looking for a quick and effortless way to transcribe your audio files or even explore transcription, without writing any code, then this solution is for you. Through an ARM template deployment, all the resources necessary to seamlessly process your audio files are set-up and set in motion.
+The simplest way to explain this tech is as a zero-touch transcription solution for all your audio files in your Azure Storage. If you are looking for a quick and effortless way to transcribe your audio files or even explore transcription, then this solution is for you. 
 
```

Comment:
I've written a new introduction to replace this entire first section. Is this correct and complete?:

This Batch Ingestion Client example transcribes all audio files found in your Azure Storage. By monitoring your Azure Storage container, this example can transcribe each new audio file automatically immediately after its upload to completes.

Reply:
yes, i will use it differently